src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -154,8 +154,7 @@ public function handle(\Exception $exception)
      *
      * @param \Exception $exception An \Exception instance
      *
-     * @see sendPhpResponse()
-     * @see createResponse()
+     * @deprecated since 2.8, to be removed and replaced by a direct call to sendPhpResponse in 3.0.
```

Comment:
deprecating a private method looks weird. You should describe what is deprecated exactly